card/mod/02_basic_types/set/all/rss.rb

```diff
@@ -49,7 +49,7 @@ def raw_feed_items args
 
 
   view :feed_title do |args|
-    Card.setting(:title) + " : " + card.name.gsub(/^\*/,'')
+    Card.global_setting(:title) + " : " + card.name.gsub(/^\*/,'')
```

Comment:
Space missing after comma.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.